packages/itensor/quri_parts/itensor/estimator.py

```diff
@@ -0,0 +1,204 @@
+import os
+from collections.abc import Collection, Iterable, Sequence
+from typing import TYPE_CHECKING, Any, Callable, NamedTuple, Optional, Union
+
+import juliacall
+from juliacall import Main as jl
+from typing_extensions import TypeAlias
+
+from quri_parts.core.estimator import (
+    ConcurrentParametricQuantumEstimator,
+    ConcurrentQuantumEstimator,
+    Estimatable,
+    Estimate,
+    ParametricQuantumEstimator,
+    QuantumEstimator,
+    create_parametric_estimator,
+)
+from quri_parts.core.operator import zero
+from quri_parts.core.state import CircuitQuantumState, ParametricCircuitQuantumState
+from quri_parts.core.utils.concurrent import execute_concurrently
+
+from .circuit import convert_circuit
+from .operator import convert_operator
+
+if TYPE_CHECKING:
+    from concurrent.futures import Executor
+
+abs_dir = os.path.dirname(os.path.abspath(__file__))
+library_path = os.path.join(abs_dir, "library.jl")
+include_statement = 'include("' + library_path + '")'
+jl.seval(include_statement)
+
+
+class _Estimate(NamedTuple):
+    value: complex
+    error: Union[float, None] = None
```

Comment:
Sorry for the confusion but now I think this is better
```suggestion
class _Estimate(NamedTuple):
    value: complex
    error: float = np.nan
```

You can follow the implementation
https://github.com/QunaSys/quri-parts/blob/c335623bce3a498b7c9a6e416ca8abbc95557317/packages/algo/quri_parts/algo/mitigation/cdr/cdr.py#L39-L41